pkg/controllers/cluster/util.go

```diff
@@ -43,6 +62,31 @@ type KubeConfigHelper struct {
 	ClientKeyData  []byte
 }
 
+type Claims struct {
```

Comment:
Only used within this util file.
```suggestion
type claims struct {
```